app/models/run.rb

```diff
@@ -45,8 +45,8 @@ class Run < ApplicationRecord
   class << self
     def programs
       [
-        LiveSplit, Llanfair, LlanfairGered, Llanfair2, WSplit, TimeSplitTracker, Portal2LiveTimer, Urn, SplitterZ,
-        Splitty, FaceSplit, ShitSplit
+        SplitsIO, LiveSplit, Llanfair, LlanfairGered, Llanfair2, WSplit, TimeSplitTracker, Portal2LiveTimer, Urn,
+        SplitterZ, Splitty, FaceSplit, ShitSplit
```

Comment:
worstrun and SourceLiveTimer are missing here?

Reply:
Thanks, forgot to add them here.